src/modules/flight_mode_manager/tasks/Transition/FlightTaskTransition.cpp

```diff
@@ -100,6 +75,13 @@ bool FlightTaskTransition::activate(const trajectory_setpoint_s &last_setpoint)
 	return ret;
 }
 
+bool FlightTaskTransition::updateInitialize()
```

Comment:
I used the opportunity to do some minor refactoring of the class. If definitions were moved around, it's probably because they were not following the same order as in the header file. I hope this is ok since the amount of changes is extremely minor.

Reply:
It's good that you take care of refactors like that, I just ask you to put it in different commits next time.